portal-api/test-helpers/api-test-helpers/database/database.config.js

```diff
@@ -0,0 +1,12 @@
+const dotenv = require('dotenv');
+
+dotenv.config();
+
+const dbName = process.env.MONGO_INITDB_DATABASE;
+// The below is different than MONGOD_URI env var as mock-data-loader does not run within the container
```

Comment:
I think this means that this is a different value from the `MONGODB_URI` environment variable in the root-level portal-api .env file (i.e., there'll be another .env file within `portal-api/test-helpers` with a different value for this env var). If I'm right in thinking this, I would change comment to something like:

`// The below value is different in the portal-api/test-helpers .env file compared to the root-level portal-api .env file as the mock-data-loader does not run within the container`

Reply:
I've updated this -- hopefully makes sense